CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+v1.11.2
```

Comment:
`Client` and `ClientContext` don't implement `LoggerAwareInterface` anymore. I think this is a breaking change, and as such it shouldn't be released in a patch release.